app/models/user.rb

```diff
@@ -154,6 +154,22 @@ def accessible_users
     end
   end
 
+  def self.taggable_for(client)
+    users = User.where(role_type: [AdminRole::TYPE, ClientSuccessRole::TYPE, GreeterRole::TYPE])
+    coalition = client.vita_partner&.coalition || client.vita_partner&.parent_organization&.coalition
```

Comment:
nit: i think this would read better if it were based on `client.vita_partner.organization?` / `client.vita_partner.site?` (same thing, just expressed differently)